packages/wearable_rotary/example/lib/main.dart

```diff
@@ -0,0 +1,59 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+
+import './custom_page_view.dart';
+
+void main() => runApp(
+      MaterialApp(
+        title: 'Rotary example app',
+        home: MyApp(),
+      ),
+    );
+
+class MyApp extends StatelessWidget {
+  final CustomPageView _horizontalPageView = CustomPageView(Axis.horizontal);
+  final CustomPageView _verticalPageView = CustomPageView(Axis.vertical);
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Rotary example app'),
+      ),
+      body: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            RaisedButton(
```

Comment:
RaisedButton is deprecated.

The linter says:

'RaisedButton' is deprecated and shouldn't be used. Use ElevatedButton instead. See the migration guide in flutter.dev/go/material-button-migration-guide). This feature was deprecated after v1.25.0-8.1.pre..
Try replacing the use of the deprecated member with the replacement.

If you want, you can enable the linter analysis by following https://github.com/flutter-tizen/plugins/pull/112#discussion_r639666218.